models/project_repository.go

```diff
@@ -0,0 +1,188 @@
+package models
+
+import (
+	"log"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/project"
+	"github.com/jinzhu/gorm"
+	satoriuuid "github.com/satori/go.uuid"
```

Comment:
Most other places imports this as "uuid". We don't really use the goa uuid package anywhere.